libcontainer/standard_init_linux.go

```diff
@@ -44,7 +44,15 @@ func (l *linuxStandardInit) getSessionRingParams() (string, uint32, uint32) {
 // the kernel
 const PR_SET_NO_NEW_PRIVS = 0x26
 
-func (l *linuxStandardInit) Init(s chan os.Signal) error {
+func (l *linuxStandardInit) Init() error {
+	waitForExec := make(chan struct{}, 1)
+	go func() {
+		if _, err := syscall.Openat(l.rootfd, execFifoFilename, os.O_WRONLY|syscall.O_CLOEXEC, 0); err != nil {
```

Comment:
We should close this `fd` before `exec`ing the user's process. No need to give them access to somewhere in `/`.